README-sitedev.md

```diff
@@ -1,41 +1,21 @@
-This documentation is for developers of this prototype OBO Foundry site.
+# Site Development
 
-Note this document is in markdown and is best viewed on GitHub.
+This documentation is for developers of this prototype OBO Foundry site.
 
 ## Getting Started
 
-You will first need jekyll, which can be installed by the ruby [gem](https://rubygems.org/):
-
-    gem install jekyll
-
-After checking out the code from GitHub, you can start a local server:
-
-    jekyll serve
-
-(this must always be done from the top level)
-
-Then view http://127.0.0.1:4000
-
-You can make changes locally (you will need to start jekyll again - no
-hot fixes).
-
-If you commit and push, your change will be visible within a few
-minutes on: http://obofoundry.github.io
-
-If this site becomes official we may want to institute policies for
-the site: e.g. major new changes happen on forks/branches, with a
-voting policy for merging these in.
-
-### Serving with Docker
-
 Because Jekyll can be difficult to install, Docker provides an
-alternative for running the `serve` command:
+alternative for running the `serve` command, then open http://127.0.0.1:4000:
```

Comment:
I agree with this, but.. ok.. Ok :D Alright alright I will wave it through.

Reply:
Haha. No humour. We have a bit of a anti-docker sentiment across the community, but it is my experience that there is almost no realistic way you can install all the local requirements on a new system now anyways. Hence - ok to delete the non-docker instructions.

Reply:
I have literally never been able to get it to work. I must be stupid because I have had problems with ruby gems in other places too.

Reply:
me neither, and i tried hard as well..